core/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -624,17 +619,13 @@ public WarmerStats warmerStats() {
         return shardWarmerService.stats();
     }
 
-    public FilterCacheStats filterCacheStats() {
-        return shardFilterCache.stats();
-    }
+    public FilterCacheStats filterCacheStats() { return indicesFilterCache.getStats(shardId); }
```

Comment:
I personally don't mind having single-line bodies on the same line as the method definition but I know some other people would, so let's just indent it as usual?